apps/frontend/src/main/index.ts

```
@@ -25,8 +25,10 @@ for (const envPath of possibleEnvPaths) {
   }
 }
 
-import { app, BrowserWindow, shell, nativeImage, session, screen } from 'electron';
+import { app, BrowserWindow, shell, nativeImage, session, screen, protocol } from 'electron';
 import { join } from 'path';
+import { readFileSync } from 'fs';
+import { URL } from 'url';
 import { accessSync, readFileSync, writeFileSync, rmSync } from 'fs';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'readFileSync'. Consider to delete it or rename it.

'readFileSync' is defined here:


(lint/suspicious/noRedeclare)